src/riak_kv_get_core.erl

```diff
@@ -284,6 +288,36 @@ num_pr(GetCore = #getcore{num_pok=NumPOK, idx_type=IdxType}, Idx) ->
             GetCore
     end.
 
+%% @private Print a warning if objects are not equal. Only called on case of no read-repair
+%% This situation could happen with pre 2.1 vclocks in very rare cases. Fixing the object
+%% requires the user to rewrite the object in 2.1+ of Riak. Logic is disabled when using
+%% a defined version of aae hashing as it's no longer needed.
+maybe_log_old_vclock(Results) ->
+    case riak_core_capability:get({riak_kv, object_hash_version}, undefined) of
+        undefined ->
+            ok;
+        0 ->
+            Version = riak_kv_entropy_manager:get_version(),
+            case [RObj || {_Idx, {ok, RObj}} <- Results] of
+                [] ->
+                    ok;
+                [_|[]] ->
```

Comment:
Very small nitpick, but the pattern I normally see to match a singleton list is just `[_]`. I'm sure this works just as well, but feels a little bit unidiomatic.